src/main/java/uk/gov/pay/adminusers/service/ExistingUserInviteCompleter.java

```diff
@@ -46,14 +45,14 @@ public Optional<InviteCompleteResponse> complete(String inviteCode) {
                                     InviteCompleteResponse response = new InviteCompleteResponse(inviteEntity.toInvite());
                                     response.setUserExternalId(userEntity.getExternalId());
                                     response.setServiceExternalId(inviteEntity.getService().getExternalId());
-                                    return Optional.of(response);
+                                    return response;
                                 } else {
                                     throw internalServerError(format("Attempting to complete user subscription to a service for a non existent service. invite-code = %s", inviteEntity.getCode()));
                                 }
-                            }).orElseGet(() -> {
+                            }).orElseThrow(() -> {
                                 throw internalServerError(format("Attempting to complete user subscription to a service for a non existent user. invite-code = %s", inviteEntity.getCode()));
                             });
```

Comment:
`orElseThrow` with a throw inside seems a little confusing to me -- that might just be me! 

Does this work?

```suggestion
                            }).orElseThrow(() -> internalServerError(format("Attempting to complete user subscription to a service for a non existent user. invite-code = %s", inviteEntity.getCode()));
```

Reply:
No, this is definitely me, failing to Java!  Will fix.